src/styles/style.css

```diff
@@ -0,0 +1,250 @@
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family 'Roboto' should be enclosed in quotes if it contains spaces or special characters. However, since 'Roboto' does not, this is not a critical issue, but it's good practice to be consistent.